core/shared/src/main/scala/fs2/internal/FreeC.scala

```diff
@@ -35,6 +35,15 @@ private[fs2] sealed abstract class FreeC[F[_], +R] {
       }
     }
 
+  def append[R2](post: => FreeC[F, R2]): FreeC[F, R2] =
```

Comment:
What do you think about calling this `andThen`?

Reply:
Well, I would certainly avoid that word, since that is the composition method from the `Function` class of the Scala library. After which the similar method in `cats.data.Kleisli` takes its cue.